src/domain/tasks/AreaDropdownButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import { Area } from 'types/area'
+
+interface areasNamesAndPks {
+  name: string
+  pk: number
+}
+interface Props {
+  areas: areasNamesAndPks[]
+  addAreaFilter: (areaPk: number) => void
+}
+const AreaDropdownButton = ({ areas, addAreaFilter }: Props): JSX.Element => {
+  const [isOpen, setIsOpen] = useState(false)
+
+  return (
+    <div className="flex justify-center">
+      <div className="flex flex-row ">
+        <button onClick={(): void => setIsOpen(true)} className="block overflow-hidden">
+          <i className="icon-arrow-right text-disabled text-sm md:text-2xl"></i>
+        </button>
+        <div className="underline">Todas las areas</div>
+      </div>
+      {isOpen && (
+        <div className="bg-disabled rounded-lg w-44 mt-4 text-sm md:text-base">
```

Comment:
We dont need a fix width here, we just need it to be full width:
![image](https://user-images.githubusercontent.com/14185059/134818031-871c412d-b86f-4317-bba0-5107846e58ed.png)